README.md

```diff
@@ -25,6 +25,7 @@ See [Getting Device Info](#getting-device-info) to determine if a device is supp
 * Service to enable "Follow Me" function.<sup>2</sup>
 * Translations
   * English
+  * Magyar

```

Comment:
@iscsu
I've been trying to the language's native name. Is this appropriate for Hungarian or is there another preferred term?

Reply:
Yes, "Magyar" is the exact and right term. Thanks! :)

Reply:
Great. Thanks for confirming.